Update slurm requirements (#98)
noaa-oar-arl/canopy-app: python/README.md

   ```
   python global_data_process.py 2020070112000,2020070112001,2020070112002,2020070112003,2020070112004
   ```
### Running global data process and canopy-app with global data (following Example 1 above)

1. Edit namelist (`namelist.canopy`) for correct number of global lat/lon grid points

   For example:
   ```
   nlat        =  1536
   nlon        =  3072
   ```

2. Edit namelist (`namelist.canopy`) for correctly named global input files and output prefix

   For example:
   ```
   file_vars    = 'input/gfs.t12z.20220630.sfcf023.canopy.nc' 'input/gfs.t12z.20220701.sfcf000.canopy.nc' 'input/gfs.t12z.20220701.sfcf001.canopy.nc'
   file_out     = 'output/2022-07-01-11-0000_global'
   ```

3. Edit namelist (`namelist.canopy`) for correct time start/end and interval

   For example:
   ```
   time_start  = '2022-07-01-11:00:00.0000'
   time_end    = '2022-07-01-13:00:00.0000'
   ntime       =  3
   time_intvl  =  3600   !!! For hourly time steps
   ```
   `time_intvl = 86400` for daily time steps (i.e., 24*3600).

5. Run global process and canopy-app

>    Running canopy-app globally requires a lot of memory, so suggest not running on head node. Slurm batch script suggestion (cpu time = ~20-30 min for a single time step):

Reviewer: ```suggestion
   Running canopy-app globally requires a lot of memory, so suggest not running on head node. Slurm batch script suggestion (cpu time = ~20-30 min per model time step):
```
